Makefile

```diff
@@ -5,7 +5,7 @@ BINARY_NAME      = nri-$(INTEGRATION)
 INTEGRATIONS_DIR = /var/db/newrelic-infra/newrelic-integrations/
 CONFIG_DIR       = /etc/newrelic-infra/integrations.d
 GO_FILES        := ./src/
-GOFLAGS			 = -mod=mod
+GOFLAGS			 = -mod=readonly
```

Comment:
could you add some comment to this ? it might be handy to have there for troubleshooting , looks like is related to the tools approach but i don't know exactly why is needed. Thanks!